deploy/templates/nstemplatetiers/advanced/cluster.yaml

```
@@ -18,20 +18,108 @@ objects:
         requests.storage: 15Gi
         requests.ephemeral-storage: 7Gi
         count/persistentvolumeclaims: "5"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-deployments
+  spec:
+    quota:
+      hard:
+        count/deployments.apps: "30"
+        count/deploymentconfigs.apps: "30"
         count/pods: "50"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-replicas
+  spec:
+    quota:
+      hard:
         count/replicasets.apps: "30"
         count/replicationcontrollers: "30"
-        count/deployments.apps: "30"
-        count/deploymentconfigs.apps: "30"
-        count/daemonsets.apps: "30"
-        count/statefulsets.apps: "30"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-routes
+  spec:
+    quota:
+      hard:
+        count/ingresses.extensions: "10"
+        count/routes.route.openshift.io: "10"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-jobs
+  spec:
+    quota:
+      hard:
         count/jobs.batch: "30"
+        count/daemonsets.apps: "30"
         count/cronjobs.batch: "30"
-        count/buildconfigs.build.openshift.io: "10"
-        count/routes.route.openshift.io: "10"
-        count/ingresses.extensions: "10"
+        count/statefulsets.apps: "30"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-services
+  spec:
+    quota:
+      hard:
         count/services: "10"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-bc
+  spec:
+    quota:
+      hard:
+        count/buildconfigs.build.openshift.io: "10"
```

Let's maybe increase it to 30 so it matches the deployment count.
```suggestion
        count/buildconfigs.build.openshift.io: "30"
```